lib/knapsack_pro/config/ci/heroku.rb

```diff
@@ -0,0 +1,31 @@
+module KnapsackPro
+  module Config
+    module CI
+      class Heroku < Base
+        def node_total
+          # not provided
+        end
+
+        def node_index
+          # not provided
+        end
+
+        def node_build_id
+          ENV['HEROKU_TEST_RUN_ID']
+        end
+
+        def commit_hash
+          ENV['HEROKU_TEST_RUN_COMMIT_VERSION']
+        end
+
+        def branch
+          ENV['HEROKU_TEST_RUN_BRANCH']
+        end
+
+        def project_dir
+          '/app' if node_build_id
```

Comment:
I assume the project directory is always in `/app` directory on Heroku CI.

Reply:
It is as far as I know, and `.git` is not present.

Reply:
@mikescar Ok. Actually, we don't need to care about `project_dir` when Heroku provides branch and git commit via ENV vars. I use `project_dir` only when I need to detect branch/git hash because they are not available via ENV for CI provider. In case of Heroku they expose branch/git hash so we should be good. The knapsack_pro won't use `project_dir`.